examples/web_bluetooth.html

```diff
@@ -82,7 +82,23 @@
 
 const scan = function () {
     if (PoweredUP.isWebBluetooth) {
-        poweredUP.scan(); // Start scanning for hubs
+        poweredUP.scan(); // Start scanning for known hubs
+    } else {
+        alert("Your browser does not support the Web Bluetooth specification.");
```

Comment:
Could we have this throw instead of alert? Having it throw gives users of this library the ability to catch the error and present whatever UX they see as best for their use case, instead of mandating an alert.

Reply:
This is the sample application, the alert has no impact on the users of the library.

Reply:
You're absolutely right, I didn't spot the context. Apologies!